apps/web/src/components/admin/AdminRegistrantsTable.tsx

```diff
@@ -51,8 +54,10 @@ export const AdminRegistrantsTable = () => {
     const options: FilterOptions = {
       limit,
       skip,
+      excludeWithdrawn: true,
```

Comment:
curious if this ever gets modified on the FE? can't we just set it to true on the BE? or is there plans to include the option to show withdrawn registrants